packages/app/config/webpack.common.js

```diff
@@ -359,6 +359,12 @@ module.exports = {
           name: 'static/media/[name].[hash:8].[ext]',
         },
       },
+      // https://github.com/gaearon/react-hot-loader/issues/1311
+      {
+        test: /\.js$/,
+        include: /node_modules\/react-dom/,
+        use: ['react-hot-loader/webpack'],
+      },
```

Comment:
To make sure the `MessageStripe` works with mapping over children and adding props to the `Action` child, I had to include this webpack plugin. I am by no means a webpack wizard, so please review this carefully.